css/styles.css

```diff
@@ -0,0 +1,137 @@
+body {
+    margin: 0;
+}
+.topnav {
+    background-color:rgb(25, 50, 126);
+    overflow: hidden;
+    width: 100%;
+    height: 3em;
```

Comment:
Q: explain the different units we have. when will we use each unit?

Reply:
Here about the different unit we have:
https://www.w3schools.com/cssref/css_units.asp
About the best practice:
Em or rem good for font-size, margin and paddin 
Vw, vh or % good for width and height.
But there isn't one answer which one is the best for width and height

Reply:
* em/rem are not recommended either: like pixels - its allowed only when its essential (like in max-height for example) or when u want a fixed size (like in here - it is ok). 
>> All margin and padding should be done in %/VH/VW and not in EM/REM.